ui/sitemaps.md

```diff
@@ -677,8 +677,13 @@ Generally, you can expected that valid HTML colors will be accepted (e.g. `green
 The color names above are agreed on between all openHAB UIs and are therefore your safest choice.
 Colors defined by a human-readable name may be adjusted for higher contrast, e.g. on a dark theme `black` may be displayed as white, because white has a higher contrast to the dark background compared to black.
 
-For better theming support Basic UI and the Android app also accepts the keywords `primary` and `secondary` as colors.
-They will be mapped to the corresponding colors of the UI theme.
+There are also the following keywords that can be used as colors:
+
+| Color Name | Note                      |
+|------------|---------------------------|
+| primary    | Primary theme color       |
+| secondary  | Secondary theme color     |
+| itemValue  | The state of a color item |
```

Comment:
You could also mention that this color will be ignored if the sitemap element has no item defined or if the item state is not a color.